### ISSUE TITLE
Change English in 700 $4 in Secondary literature

### ISSUE THREAD

**Issue by jenniferward** [rism-ch/muscat#606]
In the Secondary literature template, please change the drop-down list in 700 $4 as follows.

Please change the terminology:
Change Author to Additional author

Please change the order to the following:
Additional author 
Editor 
Composer 
Associated name
Other function

**Comment by xhero** [comment]
I can change the label, but the list is sorted by default alphabetically. If I change the sorting then all the lists in all popups will be changes and will require manual ordering. I can do it, but then the configuration must be updated everywhere.

**Comment by jenniferward** [comment]
Thanks for changing the label!

I will leave it up to you whether to change the sort order here. It is not a priority for me. (There are other lists where the order is more important; I'll send you those soon).

**Comment by jenniferward** [comment]
Update: We decided to wait on changing the order on this list and other lists for now.
